app/views/ChangeAvatarView/AvatarSuggestion.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState, useEffect } from 'react';
+import { Text, View, TouchableOpacity } from 'react-native';
+
+import { IAvatar, IUser } from '../../definitions';
+import { Services } from '../../lib/services';
+import I18n from '../../i18n';
+import Avatar from '../../containers/Avatar';
+import styles from './styles';
+import { useTheme } from '../../theme';
+
+const Item = ({
+	item,
+	onPress,
+	text,
+	testID
+}: {
+	item?: IAvatar;
```

Comment:
Are you sure something that comes straight from the rest api is going to have the exact same interface of IAvatar?